### DISCUSSION TITLE
评南京周末游

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#16] [Announcements]
# 评南京周末游

地铁 我国江浙沪地区的地铁基本架构设计是比较科学合理的，B1 站厅层，B2 站台层，两个方向的轨道环抱站台层，中间是站台，如此一来，整个地铁站形状

https://zexwoo.blog/posts/gossip/trip-in-nanjing/

**Comment by DragonOnGit** [comment]
最近南京突然就成网红城市了不知道咋回事
不过说实话，网上的东西真的只能信一半，小红书爆款真就未必是最好的🤣🤣🤣🤣
